deployment/k8s/frequency-gateway/values.yaml

```diff
@@ -53,6 +53,12 @@ nodeSelector: {}
 tolerations: []
 affinity: {}
 
+# Support vault integration
+vault:
+  enabled: false 
+  secretsPath: "secret/frequency-gateway"
+  role: "frequency-gateway-role"
+
```

Comment:
Assuming a secret store with vault is created and deployment will pull secrets from the store , currently using basic auth so role isnt used. But these CRDs offer various auth mechanism between provider and secret store